### DISCUSSION TITLE
Bruk av informasjonskapsler

### DISCUSSION THREAD

**Discussion by Murstam** [fremtind/jokul#2286] [Annet]
Vi er pliktig til å informere om bruk av informasjonskapsler og tracking og har laget et forslag til bruk på BM. Dette forslaget er i stor grad bygd på utredning og implementering gjort av SB1. 

De fleste av brukerne kommer fra SB1 og har allerede svart på dette, men vi har for øyeblikket ikke tilgang til denne informasjonen og er derfor nødt til å spørre på nytt når de kommer til oss. 

De er nødt til å svare på dette så vi har derfor valgt en pop up. 

Skisse: https://www.figma.com/file/oYhS8j3xmU0C4Xv30Dcfj8/Cookie-consent?node-id=76%3A4

**Comment by Steinop** [comment]
@Murstam : Skal Figma-skissene justeres etter innspillene fra designkritikken 05.06.20?
Assigner deg til oppgaven.

**Comment by Murstam** [comment]
@Steinop Har gjort noen endringer basert på det. Noe spesielt du tenkte på?

**Comment by Steinop** [comment]
@Murstam Var bare Figma-lenken som virket veldig lik skissen vi hadde kritikk på. Derfor usikker på om dette var en endelig versjon (klar til utvikling) eller et designutkast 😁

**Comment by joms** [comment]
@Murstam Burde ikke denne leve videre med et mål om å implementere en modal for informasjonskapsler, sporing og lignende i Jøkul?

**Comment by Murstam** [comment]
Jo, det er jo i utgangspunktet noe flere har behov for. Kan åpne den igjen. Hva blir neste steg da?

**Comment by joms** [comment]
Jeg tenker vi må finne ut om modalen vi har lagd på BM dekker andres behov på en god måte - både visuelt og teknisk. Den var vel gjennom design-kritikk om jeg husker riktig?

Kanskje et godt neste steg er å kopiere over koden fra BM og lage en Work In Progress pull request for å sende rundt til teamene for innspill.
